Dockerfile

```diff
@@ -5,13 +5,12 @@ RUN mkdir -p /data/db2 \
     && chown -R mongodb:mongodb /data/db2
 
 COPY data/ /temp/
+COPY scripts/populate-search-fields.js /temp/
```

Comment:
Not sure the scripts folder is the best place for the mongodb script as everything else is a bash script run locally, rather than in a container. 
Maybe have a 'db-scripts' folder?